thrifty-schema/src/main/java/com/microsoft/thrifty/schema/Constant.java

```diff
@@ -258,6 +258,12 @@ public void validate(Linker linker, ThriftType expected, ConstValueElement value
                 } else {
                     throw new IllegalStateException("bad enum literal: " + value.value());
                 }
+            } else if (named instanceof Typedef) {
```

Comment:
What do you mean "don't look up a typedef"? Do you mean don't classify them as enums? What validation would typdef enums have at all then?